program/src/processor/trifle/transfer_in.rs

```diff
@@ -121,6 +121,16 @@ pub fn transfer_in(
             .key;
 
         constraint_model.validate(&collection_key, &args.slot)?;
+    } else if let EscrowConstraintType::FirstCreator(_) = constraint.constraint_type {
+        let asset_data = attribute_metadata.clone().into_asset_data();
+        let creators = asset_data
+            .creators
+            .clone()
+            .ok_or(TrifleError::InvalidFirstCreator)
+            .unwrap();
```

Comment:
We discourage using unwrap in contracts because panics in the Solana Runtime don't have helpful errors. I know it's annoyingly verbose, but can you add additional checks like unwrap_or or matches on the resulting data?

Reply:
using match instead